Lib/test/test_type_aliases.py

```diff
@@ -0,0 +1,155 @@
+import types
+import unittest
+from test.support import check_syntax_error
+
+from typing import Callable, TypeAliasType, get_args, get_origin
+
+from .test_type_params import run_code
+
+
+class TypeParamsInvalidTest(unittest.TestCase):
+    def test_name_collision_01(self):
+        check_syntax_error(self, """type TA1[A, **A] = None""", "duplicate type parameter 'A'")
+
+    def test_name_non_collision_02(self):
+        run_code("""type TA1[A] = lambda A: None""")
+
+    def test_name_non_collision_03(self):
+        run_code("""\
```

Comment:
nit: since `run_code` uses `textwrap.dedent`, I don't think you need this backslash. Same in other tests in this file.
```suggestion
        run_code("""
```

Reply:
Done for all tests in test_type_aliases and test_type_params